exes/mqtt-bridge/inc/tfc_to_external.hpp

```diff
@@ -85,6 +98,10 @@ class tfc_to_external {
     spb_variables_.reserve(signals.size());
 
     for (const auto& signal : signals) {
+      if (!is_publish_signal(signal.name)) {
```

Comment:
if the list is empty should we publish everything?

Reply:
If we do that I suspect that the feature would be not used, and CPU load would always be higher than strictly necessary. If nothing is published for an empty list then the user is forced to insert the necessary signals.